### PR TITLE
Make sure a display doesn't keep a reference to a destroyed bitmap

### LINKED ISSUE #1061: Intermittent crash in ex_mouse_cursor
On MacOS I am seeing intermittent crashes with ex_mouse_cursor. When I recompile with the Address Sanitiser it tells me there is a use-after-free.
Briefly:
```
==5065==ERROR: AddressSanitizer: heap-use-after-free on address 0x61400004e5d0 at pc 0x00010129ce31 bp 0x70000e5bc650 sp 0x70000e5bc648
READ of size 8 at 0x61400004e5d0 thread T3
2019-05-27 21:14:13.583029+0100 atos[5068:195577] examining /Users/USER/*/ex_mouse_cursor [5065]
    #0 0x10129ce30 in _al_ogl_finalize_fbo ogl_fbo.c:510
```
and 
```
0x61400004e5d0 is located 400 bytes inside of 424-byte region [0x61400004e440,0x61400004e5e8)
freed by thread T3 here:
    #0 0x1001312fd in wrap_free (libclang_rt.asan_osx_dynamic.dylib:x86_64+0x592fd)
    #1 0x10120c9dd in al_free_with_context memory.c:57
    #2 0x10111582b in al_destroy_bitmap bitmap.c:248
```

### PR DESCRIPTION
**Pull request by pedro-w**
Proposed change to fix #1061 - review needed!

### CODE REVIEW

File: src/opengl/ogl_bitmap.c [hunk 1]
```diff
@@ -567,15 +567,21 @@ static void ogl_destroy_bitmap(ALLEGRO_BITMAP *bitmap)
 {
    ALLEGRO_BITMAP_EXTRA_OPENGL *ogl_bitmap = bitmap->extra;
    ALLEGRO_DISPLAY *disp;
+   ALLEGRO_DISPLAY *thisdisp;
```
Comment (SiegeLord) [inline src/opengl/ogl_bitmap.c]: Would you mind naming this 'bmp_disp' to clarify what it is?

**Comment by SiegeLord** [comment]
This change looks great, but seems too simple given the discussion in the bug, is this really all that's needed? One thing I'd like to verify for myself is that the bitmap reference is only stored in the display that owns it (in which case this change is sufficient).

**Comment by SiegeLord** [comment]
I've spent more time looking at the code, and I'm more convinced that this change is probably correct, the `opengl_target` can only be set by the display that owns the bitmap.

File: src/opengl/ogl_bitmap.c (hunk 1 above)
Comment (pedro-w) [reply src/opengl/ogl_bitmap.c]: will do - I remember thinking at the time that `thisdisp` wasn't a good name for it...

**Comment by pedro-w** [comment]
> This change looks great, but seems too simple given the discussion in the bug, is this really all that's needed? 
It does seem simple - I tried to justify it by comparing the analogous code in the directx driver but they don't really line up enough to make that approach useful.

**Comment by SiegeLord** [comment]
Thanks a lot!

**Pull request merged**